challenges/backend/backend.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask, jsonify, request
+import json
+app = Flask(__name__)
+
+data_list = []
+
+# TODO: better (more verbose) errors
+@app.route('/data', methods=['GET', 'POST', 'PATCH'])
+def data_api():
+    global data_list
+    if request.method == 'GET':
+        return jsonify(data_list)
+    elif request.method == 'POST':
+        in_json = request.get_json()
+        is_num = lambda x: type(x) (int, float), in_json
+        if len(in_json) == 500 and all(map(is_num)):
+            data_list = list(sorted(in_json))
+            return jsonify(data_list)
+        else:
+            return 'Bad request', 400
+    elif request.method == 'PATCH':
+        try:
+            data_list = list(sorted([int(request.data)] + data_list))
+            return jsonify(data_list)
+        except BaseException as e:
```

Comment:
How did you decide that `BaseException` is the correct exception to use?

Reply:
It's not ideal to handle exceptions in this manner, but I also cannot think of a situation in which it will behave in an unintended manner. I do notice now though that I didn't do sufficient testing with floats, because those won't work properly here.